ragna/deploy/_ui/central_view.py

```diff
@@ -186,20 +187,49 @@ def on_click_chat_info_wrapper(self, event):
         if self.on_click_chat_info is None:
             return
 
-        pills = "".join(
-            [
-                f"""<div class='chat_document_pill'>{d['name']}</div>"""
-                for d in self.current_chat["metadata"]["documents"]
-            ]
-        )
+        # see _api/schemas.py for `input` type definitions
```

Comment:
Display in right sidebar depending on what has been used as `input` for the chat.